src/frontend/components/SøknadsSteg/EøsSteg/Barn/useEøsForBarn.tsx

```diff
@@ -365,23 +373,23 @@ export const useEøsForBarn = (
     );
     const omsorgspersonPågåendeSøknadFraAnnetEøsLand = useJaNeiSpmFelt({
         søknadsfelt: omsorgsperson?.pågåendeSøknadFraAnnetEøsLand,
-        feilmeldingSpråkId: 'eøs-om-barn.omsorgsperson-barnetrygd-søknad.feilmelding',
-        feilmeldingSpråkVerdier: { barn: gjeldendeBarn.navn },
+        feilmelding: eøsForBarnTekster.paagaaendeSoeknadYtelseOmsorgsperson.feilmelding,
+        flettefelter: flettefelter,
         skalSkjules: borMedOmsorgsperson.verdi !== ESvar.JA,
     });
 
     const omsorgspersonPågåendeSøknadHvilketLand = useLanddropdownFeltMedJaNeiAvhengighet({
         søknadsfelt: omsorgsperson?.pågåendeSøknadHvilketLand,
-        feilmeldingSpråkId: 'eøs-om-barn.omsorgsperson-barnetrygd-hvilketland.feilmelding',
+        feilmelding: eøsForBarnTekster.hvilketLandSoektYtelseOmsorgsperson.feilmelding,
+        flettefelter: flettefelter,
```

Comment:
og her, kun ytelse men det skjer av seg selv, trenger ikke sende med flettefelt for den